.ci/conformance/tools/dbscan/download_test.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+echo
+echo "Test dbscan download started."
+echo
+wget -O test_dbscan.py https://raw.githubusercontent.com/scikit-learn/scikit-learn/master/sklearn/cluster/tests/test_dbscan.py
```

Comment:
This means that here you will use head of master instead of particular version as done in other places?